### ISSUE TITLE
Abnormal facial expressions

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#7311]
**Preferred term label:**
Abnormal facial expressions

**Synonyms**

**Definition (free text, please give PubMed ID)**
Executions of facial expressions that are visually different (ie, using different parts of the face), less accurate (ie, socially incongruous), or lower in quality (as rated by typical observers/experimenters) rather than
simply lower intensity. PMID: 32471602

**Parent term (use hpo.jax.org/app)**
Impaired use of nonverbal behaviors
HP:0000758

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-4422-1708

**Comment by pnrobinson** [comment]
This term needs work -- https://hpo.jax.org/app/browse/term/HP:0005346 (child terms are synonymous, does not belong as a morph abn of the face)

**Comment by pnrobinson** [comment]
The behavioral term was added as Abnormal production of facial expressions
Close this issuue after looking at above comment

**Comment by pnrobinson** [comment]
added
